frontend/src/app/commonComponents/Modal.tsx

```diff
@@ -27,8 +28,13 @@ const variantIcons: Record<ModalVariant, IconDefinition> = {
   warning: faExclamationCircle,
 };
 
-const Header: React.FC<{}> = ({ children }) => (
-  <h3 className="modal__heading">{children}</h3>
+const Header: React.FC<{ styleClassNames?: string }> = ({
+  children,
+  styleClassNames,
+}) => (
+  <h1 className={"modal__heading font-sans-lg " + styleClassNames}>
```

Comment:
It seems like there's a slight difference in font size with the new version - it's a little smaller. It's not a big difference so probably fine but might be worth checking.

Reply:
I'll check with Kenny later today to see what his thoughts are.